src/app/components/PlayerPanel.ts

```diff
@@ -109,37 +109,85 @@ export class PlayerPanel {
           'player-panel__image',
         ),
         `x${count}`,
-      ),
+      ),*/
+  // );
+  // }
+  private createResultTitle(): HTMLElement {
+    return Render.elementFactory(
+      'p',
+      { className: 'player-panel__title' },
+      'Results',
     );
   }
-
   private createResultWindow(): HTMLElement {
-    return Render.elementFactory(
-      'div',
-      {
-        className: 'player-panel__result',
-      },
-      'Results:',
-    );
+    return Render.elementFactory('div', {
+      className: 'player-panel__result',
+    });
   }
 
-  private createButtonPanel(): HTMLElement {
-    return Render.elementFactory(
-      'div',
-      { className: 'player-panel__buttons' },
-      this.createDiceButton(),
-      this.createExchangeButton(),
+  // private createButtonPanel(player: Player): HTMLElement {
+  // return Render.elementFactory(
+  // 'div',
+  // { className: 'player-panel__buttons' },
+  // this.createExchangeButton(player),
+  // this.createDiceButton(player),
+  // );
+  // }
+  private createExchangeButton(player: Player): HTMLElement {
+    const tradeBtn = Render.elementFactory(
+      'button',
+      {
+        id: 'exchange',
+        className: 'btn button',
+        style: `border:1px solid ${player.theColor}`,
```

Comment:
I think for the creation moment, it should just receive border-width and border-style, the color will be updated when the current player changes